src/garage/tf/algos/ddpg.py

```diff
@@ -352,14 +353,15 @@ def _optimize_policy(self):
             float: Q value predicted by the q network.
 
         """
-        transitions = self._replay_buffer.sample_transitions(
+        timesteps = self._replay_buffer.sample_timesteps(
             self._buffer_batch_size)
 
-        observations = transitions['observations']
-        next_observations = transitions['next_observations']
-        rewards = transitions['rewards'].reshape(-1, 1)
-        actions = transitions['actions']
-        terminals = transitions['terminals'].reshape(-1, 1)
+        observations = timesteps.observations
+        rewards = timesteps.rewards.reshape(-1, 1)
```

Comment:
hmm why does the shape differ from what's needed? you would expect TimeStepBatch to already be perfect for vanilla off-policy RL, so maybe we should change the internal array shape